docs/learn/benefits-lukso-standards.md

```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
 | Upgradeable Security           | ✅                         | ✅                          | ✅                                      | ✅ Through [LSP14](../standards/access-control/lsp14-ownable-2-step.md)                             |
 | Pre and Post Execution Hooks   | ✅                         | ✅                          | ✅                                      | ✅ Through [LSP20](../standards/accounts/lsp20-call-verification.md)                                |
 
-As shown from the table above, the [Universal Profile Browser Extension](/install-up-browser-extension) provides more features and a better user experience when interacting with dApps. Universal Profiles (known as 🆙) are smart contract accounts that offer many benefits.
+| Feature                                        | Benefits                                                                                                                                                                                                                                                                                                                           |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **🗄️ Metadata in one place**                   | Finished re-filling the same information again and again on each new app you register to!<br/>Your information is stored and fetched from the same place (your 🆙 storage) and displayed the same across all the dApps ecosystem!                                                                                                  |
+| **🎨 Customizable metadata**                   | Make your Universal Profile stand out from others! Give it a profile and cover image, and add as many amount of information with its unlimited storage.                                                                                                                                                                            |
+| **⛽️ Gas-Less transactions**                  | No more need to buy and hold native tokens to pay for the gas. Use the transaction relayer plugged to the Universal Profile from the start to get started.                                                                                                                                                                         |
```

Comment:
"transaction relayer plugged to the Universal Profile from the start" - should we specify that it's only related to profiles deployed using universalprofile.cloud or at least that's what we provide?

```suggestion
| **⛽️ Gas-Less transactions**                  | No more need to buy and hold native tokens to pay for the gas. Use the transaction relayer plugged to the Universal Profile from the start to get you going.                                                                                                                                                                         |
```